From eefd81ad9716dec060afc785dac6dab3e0961aa2 Mon Sep 17 00:00:00 2001
From: Zhihang Wei <wzh@ilbers.de>
Date: Wed, 10 Dec 2025 15:39:56 +0100
Subject: [PATCH v2] dtbsinstall: fix installing DT overlays

Add dtbo target needed for __dtbs_install to fix 'make dtbs_install'
call. 'No rule to make target' error will occur otherwise.

Changes since v1:
- rebase to new kernel v6.12.20-2.0.0-phy

Signed-off-by: Anton Mikanovich <amikan@ilbers.de>
Signed-off-by: Zhihang Wei <wzh@ilbers.de>

---
 scripts/Makefile.dtbinst | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/scripts/Makefile.dtbinst b/scripts/Makefile.dtbinst
index 9d920419a62c..076cf1df8e57 100644
--- a/scripts/Makefile.dtbinst
+++ b/scripts/Makefile.dtbinst
@@ -22,6 +22,9 @@ quiet_cmd_dtb_install = INSTALL $@
 $(dst)/%: $(obj)/%
 	$(call cmd,dtb_install)
 
+$(dst)/%.dtbo: $(obj)/%.dtbo
+	$(call cmd,dtb_install)
+
 dtbs := $(patsubst $(obj)/%,%,$(call read-file, $(obj)/dtbs-list))
 
 ifdef CONFIG_ARCH_WANT_FLAT_DTB_INSTALL
-- 
2.39.5

